agent/security_helper.py

```diff
@@ -336,3 +340,32 @@ def cpu_vulnerabilities():
         res['mitigations_disabled'] = mitigations_disabled
 
     return res
+
+
+def patch_sshd_config(patch_param, safe_value):
```

Comment:
Can we backup the file first?

Reply:
It gets backed up before and if it gets changed. Look at the end of the function.